vanilla-framework/vanilla-dashboard-theme
# Accordion

## Done

- Creates and accessible accordion pattern to be used in the dashboard theme.

## QA

- Pull down branch and run `./run` 
- View the new pattern here http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/accordion/accordion/
- Check the designs here https://github.com/ubuntudesign/vanilla-dashboard-theme-design/blob/master/Accordion

## Details

- Fixes #13
